javalib/src/main/scala/java/lang/System.scala

```diff
@@ -24,12 +24,68 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
-  def clearProperty(key: String): String                = ???
-  def getProperties(): Properties                       = ???
-  def getProperty(key: String): String                  = ???
-  def getProperty(key: String, default: String): String = ???
-  def setProperty(key: String, value: String): String   = ???
+  private def loadProperties() = {
+    val sysProps = new Properties()
+    sysProps.setProperty("java.version", "1.8")
+    sysProps.setProperty("java.vm.specification.version", "1.8")
+    sysProps.setProperty("java.vm.specification.vendor", "Oracle Corporation")
```

Comment:
We do not strictly adhere to JVM requirements, only to Scala requirements. But at the same time seeing different values there might confuse the apps that previously were never run outside the reference environment. We should keep it to be "Oracle Corporation", given that we strive for reasonable source compatibility wrt JDK 8.